### PR TITLE
Added radio groups and moved some stuff from Label

### PR DESCRIPTION
**Pull request by hhaga**
Added new component `RadioGroup` to use for grouped radio buttons.
To avoid wrapping of all groups in `Label`s we moved `bottomInfo` and `topInfo` out of `Label` and into a new `Common.elm` .
The thought is to use `topInfo` and `bottomInfo` inside the individual components instead of wrapping the components in `Label` where they are used.

To clarify: `topInfo` and `bottomInfo` will still only be used inside the `elm-components library`

### CODE REVIEW

File: src/Nordea/Components/Common.elm [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                            [ String.fromInt charCounter.current ++ "/" ++ String.fromInt charCounter.max |> text ]
+                    )
+    in
+    [ div
+        [ css
+            [ displayFlex
+            , flexBasis (pct 100)
+            , justifyContent spaceBetween
+            , marginTop (rem 0.5)
+            , Css.property "gap" "1rem"
```
Comment (mathilwa) [inline src/Nordea/Components/Common.elm:84]: Can `(rem 1)` be used insted of `"1rem"`?

File: src/Nordea/Components/Common.elm [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                            ]
+                        ]
+                        [ Icons.error [ css [ marginRight (rem 0.5), flex none ] ]
+                        , text errText
+                        ]
+                )
+        , charCounterView
+        ]
+        |> showIf (Maybe.isJust config.errorMessage)
+    , div [ css [ displayFlex, flexBasis (pct 100), justifyContent spaceBetween, marginTop (rem 0.5), Css.property "gap" "1rem" ] ]
```
Comment (mathilwa) [inline src/Nordea/Components/Common.elm:105]: same here as the comment above

File: src/Nordea/Components/Label.elm [hunk 3]
```diff
@@ -113,6 +89,31 @@ init labelText labelType =
...
+                            Just (Common.Optional a)
+
+                        Just (Custom a) ->
+                            Just (Common.Custom a)
+
+                        Nothing ->
+                            Nothing
+            in
+            { labelText = config.labelText
+            , requirednessHint = config.requirednessHint |> fromLabelRequirednessHint
```
Comment (mathilwa) [inline src/Nordea/Components/Label.elm:110]: just wondering about the wording here, requirednessHint. Would requiredHint be sufficient?

File: src/Nordea/Components/Common.elm (hunk 1 above)
Comment (hhaga) [reply src/Nordea/Components/Common.elm:84]: I do not think so. As I understood it, the "gap" property is only available through this Css.property and it takes two strings as parameters and need to be written in same way as in CSS.

File: src/Nordea/Components/Label.elm (hunk 3 above)
Comment (hhaga) [reply src/Nordea/Components/Label.elm:110]: I think this is a hint regarding the requiredness of the field. I guess requiredHint would work fine as well, but maybe that was not chosen so it should not be mistaken for being a hint that is required (?) 🤔   We actually copied that type (and wording) from `Label` 😅

**Review by fredrikloberg — COMMENTED** [review]
I'm wondering whether this improves the readability, the only difference I see is the `Label.GroupLabel` param 🤔


```elm
RadioGroup.init
    "Group label"
    |> RadioGroup.withErrorMessage "Some error message"
    |> RadioGroup.view
        []
        [ RadioButton.init
            "simple"
            (text "Click me: ")
            NoOp
            |> RadioButton.withHasError True
            |> RadioButton.view []
        , RadioButton.init
            "simple"
            (text "Click me: ")
            NoOp
            |> RadioButton.withHasError True
            |> RadioButton.view []
         ]
```
Vs.
```elm
Label.init
   "Group label"
    Label.GroupLabel
    |> Label.withErrorMessage "Some error message"
    |> Label.view
        []
        [ RadioButton.init
            "simple"
            (text "Click me: ")
            NoOp
            |> RadioButton.withHasError True
            |> RadioButton.view []
        , RadioButton.init
            "simple"
            (text "Click me: ")
            NoOp
            |> RadioButton.withHasError True
            |> RadioButton.view []
        ]
```

What do you guys think? 🙂

File: src/Nordea/Components/Common.elm (hunk 1 above)
Comment (mathilwa) [reply src/Nordea/Components/Common.elm:84]: 👍

File: src/Nordea/Components/Label.elm (hunk 3 above)
Comment (mathilwa) [reply src/Nordea/Components/Label.elm:110]: 😅

**Comment by mathilwa** [comment]
> I'm wondering whether this improves the readability, the only difference I see is the `Label.GroupLabel` param 🤔
> 
> ```elm
> RadioGroup.init
>     "Group label"
>     |> RadioGroup.withErrorMessage "Some error message"
>     |> RadioGroup.view
>         []
>         [ RadioButton.init
>             "simple"
>             (text "Click me: ")
>             NoOp
>             |> RadioButton.withHasError True
>             |> RadioButton.view []
>         , RadioButton.init
>             "simple"
>             (text "Click me: ")
>             NoOp
>             |> RadioButton.withHasError True
>             |> RadioButton.view []
>          ]
> ```
> 
> Vs.
> 
> ```elm
> Label.init
>    "Group label"
>     Label.GroupLabel
>     |> Label.withErrorMessage "Some error message"
>     |> Label.view
>         []
>         [ RadioButton.init
>             "simple"
>             (text "Click me: ")
>             NoOp
>             |> RadioButton.withHasError True
>             |> RadioButton.view []
>         , RadioButton.init
>             "simple"
>             (text "Click me: ")
>             NoOp
>             |> RadioButton.withHasError True
>             |> RadioButton.view []
>         ]
> ```
> 
> What do you guys think? 🙂

I think that the biggest improvement in readability here is that its  (in my head) more logical to use radiogroup than label? 😊

**Review by mathilwa — APPROVED** [review]

**Pull request merged**